plugins/rss/src/rss.js

```diff
@@ -17,31 +17,33 @@ const rssPlugin = async (html, route) => {
         feed.addCategory(cat);
       });
     }
-    const mdString = readFileSync(route.templateFile, 'utf8').toString();
+    if (route.published !== false || route.data.published !== false) {
```

Comment:
I logged the route data and published is only available for `route.data.published`

```
{
  postRenderers: [ 'fouc', 'seoHrefOptimise', 'lazyImages', 'mediumZoom', 'rss' ],
  route: '/blog/2020-03-21-food',
  type: 'contentFolder',
  templateFile: '/Users/m/Dev/notiz/scully-plugins/scully-plugin-lazy-images/content/blog/2020-03-21-food.md',
  data: {
    name: undefined,
    title: 'Food',
    description: 'Food description',
    publishedAt: 2020-03-25T10:12:00.000Z,
    updatedAt: 2020-03-25T10:12:00.000Z,
    published: true,
    tags: [ 'lazy-images' ],
    authors: [ 'Bruce Lee' ],
    sourceFile: '2020-03-21-food.md'
  },
  config: {
    type: 'contentFolder',
    slug: { folder: './content/blog' },
    postRenderers: [ 'fouc', 'seoHrefOptimise', 'lazyImages', 'mediumZoom', 'rss' ]
  }
}
```

We can simplify the code with

```suggestion
    if (route.data.published) {
```